src/operations/operation.ts

```diff
@@ -49,21 +47,22 @@ export abstract class OperationBase<
   // BSON serialization options
   bsonOptions?: BSONSerializeOptions;
 
-  constructor(options: T = {} as T) {
-    this.options = Object.assign({}, options);
+  [kSession]: ClientSession;
```

Comment:
I think this should be `[kSession]?: ClientSession` I guess TS has some issues around symbol properties that it's not enforcing this.